catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/controller/NotificationController.java

```diff
@@ -81,41 +81,52 @@ public void handleEvent(Event event) throws IllegalArgumentException {
                 || event.getProperty(Notification.NOTIFICATION_KEY_APPLICATION)
                 .toString()
                 .isEmpty()) {
-            throw new IllegalArgumentException(
-                    "Event \"" + Notification.NOTIFICATION_KEY_APPLICATION
-                            + "\" property is null or empty");
+            String message = "Event [" + Notification.NOTIFICATION_KEY_APPLICATION
```

Comment:
@figliold should work with @oconnormi on how and who will resolve using `String.format()` which was a comment on https://github.com/codice/ddf/pull/1027